wp-content/themes/core/blocks/quote/quote.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+declare( strict_types=1 );
+
+use Tribe\Project\Blocks\Types\Quote\Quote_Model;
+
+// phpcs:ignore VariableAnalysis.CodeAnalysis.VariableAnalysis.UndefinedVariable
+$model = new Quote_Model( $args[ 'block' ] );
+
+get_template_part(
```

Comment:
Dumb, but I'm curious why you're wrapping to new lines with this when it's shorter than ~80 chars/line length limits.